### PR TITLE
[WJ-456] Add ftml-wasm package

### PR DESCRIPTION
**Pull request by Monkatraz**
~~Blocked by: https://github.com/scpwiki/wikijump/pull/195~~

This adds a package which wraps around and makes accessible FTML's WASM compile target. It makes a few compromises, unfortunately, mainly that binaries are included in repo due to the complications of introducing a Rust build into the ES module building pipeline.

### CODE REVIEW

**Comment by Monkatraz** [comment]
Note: repo. needs a rebase when https://github.com/scpwiki/wikijump/pull/204 gets merged (FTML version bump, interface changes)

File: client/modules/ftml-wasm/pkg/ftml.d.ts [hunk 1]
```diff
@@ -0,0 +1,263 @@
+/* tslint:disable */
```
Comment (emmiegit) [inline client/modules/ftml-wasm/pkg/ftml.d.ts]: Maybe rename `pkg` -> `vendor` for clarity?

**Review by emmiegit — APPROVED** [review]
This makes sense as a first version of the module.

File: client/modules/ftml-wasm/pkg/ftml.d.ts (hunk 1 above)
Comment (Monkatraz) [reply client/modules/ftml-wasm/pkg/ftml.d.ts]: That does make more sense. Let me do a rebase and I'll make that change with it.

**Comment by Monkatraz** [comment]
thanks @ammongit 

I'm gonna merge this - @rossjrw if you have issues, let me know - I'll get them tracked and fixed. But you're gonna want to save your energy for the next PR.

**Pull request merged**